### PR TITLE
- Updated url regex to require non-whitespace characters.

### PR DESCRIPTION
**Pull request by theall38103**

### CODE REVIEW

**Comment by kurtseifried** [comment]
Shouldn't we just use the special builtin "uri" to check the uri?

**Comment by kurtseifried** [comment]
closing because of newer merge

**Comment by theall38103** [comment]
Same as https://github.com/CVEProject/automation-working-group/pull/57#issuecomment-352890775 -- 'uri' would allow urls like "mailto:cve@mitre.org" or "file:///c:/WINDOWS/win.ini".

File: cve_json_schema/CVE_JSON_4.0_min.schema [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "url": {
           "maxLength": 500,
           "type": "string",
-          "pattern": "^(ftp|http)s?://[^ \t]+$"
+          "pattern": "^(ftp|http)s?://\\S+$"
```
Comment (DieNode) [inline cve_json_schema/CVE_JSON_4.0_min.schema:44]: "pattern": "^(ftp|http)s?://[^ \t]+$"

**Pull request closed without merging**